src/ClientModel.cpp

```diff
@@ -99,7 +99,7 @@ void ClientModel::init(QString _dbpath)
 	if (m_dbpath.isEmpty())
 		m_client.reset(new MixClient(QStandardPaths::writableLocation(QStandardPaths::TempLocation).toStdString()));
 	else
-		m_client.reset(new MixClient(m_dbpath.toStdString()));
+		m_client.reset(new MixClient(QStandardPaths::writableLocation(QStandardPaths::TempLocation).toStdString() + "/" + m_dbpath.toStdString()));
```

Comment:
ok cleaned up added. 
We still need to be able to specify another path cause two instances of ClientModel can run at the same time.